File: beginner/pagina01/1020/1020.cpp
#include <iostream>

using namespace std;

int main() {
	
	int value;
	cin >> value;

	int ages = value / 365;
	value %= 365;

	int months = value / 30;
	value %= 30;

	int days = value;

	cout << ages << " ano(s)\n";
	cout << months << " mes(es)\n";
	cout << days << " dia(s)\n";
	
	return 0;
}

File: beginner/pagina01/1020/1020.rs
use std::io;

fn main() {

    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    let mut value: u32 = input.trim().parse().unwrap();

    let ages = value / 365;
	value %= 365;

	let months = value / 30;
	value %= 30;

	let days = value;

	println!("{} ano(s)", ages);
	println!("{} mes(es)", months);
	println!("{} dia(s)", days);
}